Missionframework/modules/11_permission/fnc/fn_permission_import.sqf

```diff
@@ -0,0 +1,28 @@
+#include "script_component.hpp"
+/*
+    KPLIB_fnc_permission_import
```

Comment:
Same as for export.... where is it used at the moment?

Reply:
Same then export.